tests/mmm/test_delayed_saturated_mmm.py

```diff
@@ -254,6 +258,93 @@ def test_get_fourier_models_data(
             assert fourier_modes_data.max().max() <= 1
             assert fourier_modes_data.min().min() >= -1
 
+    def test_channel_contributions_forward_pass_recovers_contribution(
+        self, mmm_fitted: DelayedSaturatedMMM
+    ) -> None:
+        channel_data = mmm_fitted.X[mmm_fitted.channel_columns].to_numpy()
+        channel_contributions_forward_pass = (
+            mmm_fitted.channel_contributions_forward_pass(channel_data=channel_data)
+        )
+        channel_contributions_forward_pass_mean = (
+            channel_contributions_forward_pass.mean(axis=(0, 1))
+        )
+        channel_contributions_mean = mmm_fitted.fit_result[
+            "channel_contributions"
+        ].mean(dim=["draw", "chain"])
+        assert (
+            channel_contributions_forward_pass_mean.shape
+            == channel_contributions_mean.shape
+        )
+        np.testing.assert_array_almost_equal(
+            x=channel_contributions_forward_pass_mean / channel_contributions_mean,
+            y=mmm_fitted.y.max(),
+        )
+
+    def test_channel_contributions_forward_pass_is_consistent(
+        self, mmm_fitted: DelayedSaturatedMMM
+    ) -> None:
+        channel_data = mmm_fitted.X[mmm_fitted.channel_columns].to_numpy()
+        channel_contributions_forward_pass = (
+            mmm_fitted.channel_contributions_forward_pass(channel_data=channel_data)
+        )
+        channel_contributions_forward_pass_grid = (
+            mmm_fitted.get_channel_contributions_forward_pass_grid(
+                start=0, stop=2, num=3
+            )
+        )
+        assert channel_contributions_forward_pass_grid[0].sum().item() == 0
+        assert (
+            channel_contributions_forward_pass_grid[1].to_numpy()
+            == channel_contributions_forward_pass
+        ).all()
```

Comment:
In general I like the `np.testing.assert_allclose` more because it gives a nice output when it fails. I see you used above, so it's not really an advice :)

No need to change at all

Reply:
Actually, your suggestion is nicer and more consistent. Changed in https://github.com/pymc-labs/pymc-marketing/pull/313/commits/c48adc055c0f8471fecdc6babc3498859cf34d16